spring-boot/admin/src/main/java/eu/coatrack/admin/service/GatewayHealthMonitorService.java

```diff
@@ -69,6 +69,16 @@ public HealthDataForOneGateway(Proxy proxy) {
         }
     }
 
+    public class HealthDataForOneGatewayPlusSummaryStatus {
```

Comment:
Name is confusing, as this is _not_ for one gateway.

As it contains all data required by the GUI, maybe just call it `DataForGatewayHealthMonitor` ?

Reply:
Done, I refactored the name to DataForGatewayHealthMonitor